.gitignore

```
@@ -77,4 +77,4 @@ xcuserdata
 *.env
 
 # built docs
-public
+/public
```

Should it be? 
```suggestion
public/
```

`/public` makes me think it is going to try to ignore a "public" directory at the root of the file system, not the project.

---

No `public/` will make it ignore any folder named `public` in the project. `/public` is the only way to make it only ignore a folder named `public` in the root of the project.

---

👍